### PR TITLE
fix[HACBS-840] provide space for docker build

### PR DESCRIPTION
**Pull request by jsztuka**
Multiple reasons why the build image was not passing throught a workflow job:

Since the scale of build image exceeded 11GB, it was unable to finish the build process (no space left on device).
This problem persisted even though the enough space was provided. 
Removing `buildah` section helped to solve this issue together with adding some space within the github runner.
We also don't need to provide a support for multiple architectures at this time, hence the removal of qemu.

Maybe worth mentioning adding this github action https://github.com/marketplace/actions/free-disk-space-ubuntu
in near future.

### CODE REVIEW

**Review by dirgim — APPROVED** [review]
LGTM - it would have been nice to be able to use the existing github actions, but if they don't support our image size requirements, this change should do the job.

**Comment by jsztuka** [comment]
> LGTM - it would have been nice to be able to use the existing github actions, but if they don't support our image size requirements, this change should do the job.

Agreed, I was trying multiple ways to keep the github actions, unfortunately it turned out that they don't support this case of behavior.

File: .github/workflows/clair-in-ci-db.yaml [hunk 1]
```diff
@@ -22,38 +22,29 @@ jobs:
...
           password: ${{ secrets.HACBS_TEST_QUAY_TOKEN }}
+      
+      - name: Setup docker for workflow
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
 
       - name: Pull-and-retag-image
```
Comment (hongweiliu17) [inline .github/workflows/clair-in-ci-db.yaml:39]: This is not related to this PR, `skopeo copy` can be used to tag an image without pulling the image to local dir.

**Review by hongweiliu17 — APPROVED** [review]
LGTM
